apps/oc_chef_wm/src/oc_chef_wm_base.erl

```diff
@@ -658,7 +663,8 @@ finish_request(Req, #base_state{reqid = ReqId,
         end
     catch
         X:Y ->
-            lager:error({X, Y, erlang:get_stacktrace()}),
+            lager:error("Error: ~p:~p. Stack trace follows.", [X, Y]),
+            lager:error("Stack Trace: ~p",  [erlang:get_stacktrace()]),
```

Comment:
:+1: Were these messages even getting logged before? I seem to remember dropping a bunch of things when we moved over from `error_logger` to `lager`, and lager wouldn't blow up when you called a function that didn't exist. This looks like one of those.